src/Laravel/Commands/WebhookCommand.php

```diff
@@ -18,6 +18,10 @@
 use Longman\TelegramBot\Request;
 use PhpTelegramBot\Laravel\PhpTelegramBotContract;
 
+use function json_encode;
```

Comment:
@rsthegeek Would you mind explaining to me the benefit of these?

Reply:
Actually, it doesn't have any benefits but the rules of longman/php-code-style v7 suggested having it in order to pass phpcs checks.